package.json

```diff
@@ -62,7 +62,8 @@
     "infra": "node scripts/infra-cmds.js",
     "run-proxies": "./scripts/run-proxies.sh",
     "proxies": "yarn run-proxies",
-    "hmr-benchmark": "node -r esbuild-register scripts/hmr-benchmark.ts"
+    "hmr-benchmark": "node -r esbuild-register scripts/hmr-benchmark.ts",
+    "mock-national-registry": "mockoon-cli start --data ./mocks/national-registry.json"
```

Comment:
I was struggling with yarn.lock because I had added mockoon-cli as a dependency but apparently in my many struggles I forgot to re-add the dependency. I'll just roll back package.json and yarn.lock and add the mockoon-cli instructions to the README.